renderdoc/driver/gl/gl_common.h

```diff
@@ -264,65 +264,90 @@ extern int GLCoreVersion;
-  EXT_TO_CHECK(40, ARB_shader_subroutine)                \
-  EXT_TO_CHECK(40, ARB_tessellation_shader)              \
-  EXT_TO_CHECK(40, ARB_texture_cube_map_array)           \
-  EXT_TO_CHECK(40, ARB_transform_feedback2)              \
-  EXT_TO_CHECK(41, ARB_separate_shader_objects)          \
-  EXT_TO_CHECK(41, ARB_viewport_array)                   \
-  EXT_TO_CHECK(42, ARB_base_instance)                    \
-  EXT_TO_CHECK(42, ARB_shader_atomic_counters)           \
-  EXT_TO_CHECK(42, ARB_shader_image_load_store)          \
-  EXT_TO_CHECK(42, ARB_shading_language_420pack)         \
-  EXT_TO_CHECK(42, ARB_texture_storage)                  \
-  EXT_TO_CHECK(43, ARB_clear_buffer_object)              \
-  EXT_TO_CHECK(43, ARB_compute_shader)                   \
-  EXT_TO_CHECK(43, ARB_copy_image)                       \
-  EXT_TO_CHECK(43, ARB_ES3_compatibility)                \
-  EXT_TO_CHECK(43, ARB_internalformat_query2)            \
-  EXT_TO_CHECK(43, ARB_program_interface_query)          \
-  EXT_TO_CHECK(43, ARB_shader_storage_buffer_object)     \
-  EXT_TO_CHECK(43, ARB_stencil_texturing)                \
-  EXT_TO_CHECK(43, ARB_texture_storage_multisample)      \
-  EXT_TO_CHECK(43, ARB_texture_view)                     \
-  EXT_TO_CHECK(43, ARB_vertex_attrib_binding)            \
-  EXT_TO_CHECK(43, KHR_debug)                            \
-  EXT_TO_CHECK(44, ARB_enhanced_layouts)                 \
-  EXT_TO_CHECK(44, ARB_query_buffer_object)              \
-  EXT_TO_CHECK(45, ARB_clip_control)                     \
-  EXT_TO_CHECK(99, ARB_indirect_parameters)              \
-  EXT_TO_CHECK(99, ARB_seamless_cubemap_per_texture)     \
-  EXT_TO_CHECK(99, EXT_depth_bounds_test)                \
-  EXT_TO_CHECK(99, EXT_direct_state_access)              \
-  EXT_TO_CHECK(99, EXT_polygon_offset_clamp)             \
-  EXT_TO_CHECK(99, EXT_raster_multisample)               \
-  EXT_TO_CHECK(99, EXT_texture_swizzle)                  \
-  EXT_TO_CHECK(99, KHR_blend_equation_advanced_coherent) \
-  /* OpenGL ES extensions */                             \
-  EXT_TO_CHECK(99, EXT_clip_cull_distance)               \
-  EXT_TO_CHECK(99, EXT_multisample_compatibility)        \
-  EXT_TO_CHECK(99, NV_polygon_mode)                      \
-  EXT_TO_CHECK(99, NV_read_depth)                        \
-  EXT_TO_CHECK(99, NV_read_stencil)                      \
-  EXT_TO_CHECK(99, NV_read_depth_stencil)                \
-  EXT_TO_CHECK(99, OES_sample_shading)
+// List of extensions and the versions when they became core (first column for GL, second column for
+// GLES). In case of GLES compatible extensions and new features of the different versions are also
+// taken into account.
+// 99 means the extension never became core, so you can easily just do a check of CoreVersion >= NN
+// and they will always fail.
+// 88 means the extension is not available for the driver or it has no compatible counterpart.
```

Comment:
Actually there is no need to distinguish between these two cases. In case of the GLES extensions I just wanted to clarify that those extensions do not exist for GL. In the other case I wanted to indicate that a GL extension defines such features/functions/etc that are neither in core GLES, nor present in any similar GLES extension. If you think so, I can replace each 88 with 99.